.cloudbuild/ci/unit-tests.yaml

```diff
@@ -20,6 +20,10 @@ steps:
     args: 
       - -c
       - |
+        pushd . && mkdir -p /opt && cd /opt && curl -L https://github.com/libbpf/libbpf/archive/refs/tags/v0.6.1.tar.gz | tar xz && \
+        cd /opt/libbpf-0.6.1/src && \
+        make && \
+        make install && popd && \
```

Comment:
Hrm, I think this should be fine for tests, but just a heads up we are using a forked version of `libbpf` to maintain compatibility with CentOS 7: https://github.com/gravitational/teleport/commit/e901ce4b3f5a0caccb1e295ccabb373eeee85590.

Reply:
I saw that we have our own fork, but in my case tests were failing because my kernel were too new for our `libbpf`. I was just curious if updating it maybe changes something. It didn't.
Anyways, I reverted that change.